rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  belongs_to :book
+  belongs_to :person
+
+  attr_accessor :date
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
```

Comment:
- [ ] I think it would be great if you  let The rental is adding itself to the @book achieving the 'belongs-to' association ✨✅🔥